### ISSUE TITLE
Date columns aren't deserialized properly

### ISSUE THREAD

**Issue by danelkhen** [typeorm/typeorm#179]
Tested with sqlite driver only:
```
@Table()
export class MyRecord {
    @PrimaryColumn() key: string;
    @Column(CT.DATETIME, { nullable: true }) mtime: Date;
}

var x= {key:"gg", mtime:new Date()};
persist(x)
```
SQL
```
INSERT INTO "my_record"("key", "mtime") VALUES ($1,$2) -- PARAMETERS: ["gg","2017-1-9 11:50:4"]
```

```2017-1-9 11:50:4``` seems like a wrong value for the date parameter, But also after that - when loading the objects with typeorm the value of the property is of type string and not Date.

**Comment by Luke265** [comment]
Duplicate #177, I've also created PR #178, but i have to write tests, before this gets merged.

**Comment by danelkhen** [comment]
Thanks, I saw your PR, but it only talked about storing the data, so I wasn't sure if it includes a fix to the fact that the value is not being converted during deserialization into an object. If this is the case we can close this one...

**Comment by Luke265** [comment]
I've updated my PR which now includes date, datetime, time deserialization into date object.

**Comment by danelkhen** [comment]
👍 Great, thanks!

**Comment by pleerock** [comment]
released in `0.0.7-alpha.10` please check if everything is working as expected there
